deploy/helm/templates/deployment.yaml

```diff
@@ -71,6 +73,8 @@ spec:
               value: {{ .Values.operator.batchDeleteDelay | quote }}
             - name: OPERATOR_METRICS_BIND_ADDRESS
               value: ":8080"
+            - name: OPERATOR_METRICS_FINDINGS_ENABLED
+              value: {{ .Values.operator.metricsFindingsEnabled | quote }}
```

Comment:
I would make it in a separate PR , to not mix concerns , also update pr description

Reply:
Ideally I agree with you, but there are also other minor diffs fixed in this PR.

Reply:
because `OPERATOR_METRICS_FINDINGS_ENABLED` is related to metrics feature and If for some reason we will need to revert this PR commit we should revert the metrics features flag

Reply:
https://github.com/aquasecurity/trivy-operator/pull/179. I'll rebase this PR after merging the other one.

Reply:
Rebased